src/utils/GridWrapper.js

```diff
@@ -0,0 +1,17 @@
+import styled from '@emotion/styled';
+
+const GridWrapper = ({ children, cols }) => {
```

Comment:
Also I'd be okay with just calling the component `Grid` to keep it more concise.

Reply:
You could just create the component similar to this, using a function to change style based on props and then export it. That way you can just export the base `Grid` component you've got there without having to create a `GridWrapper` component.
https://emotion.sh/docs/styled#changing-based-on-props

Reply:
Sorry if what I said isn't super clear. I can try to explain it better later if necessary.

Reply:
It was clear! I just made the changes. It is definitely a lot cleaner! Let me know if you see anything else!